cloudfront/wellcomelibrary.org/edge-lambda/src/staticRedirects.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
How come we're committing the JSON _and_ the CSV? Couldn't we get away with just the CSV?

Reply:
I suppose you'd just have to make sure you build the assets as part-of or before the test phase.

Reply:
I would just whack it in the test/build commands and add it to the .gitignore...